tests/common/config_reload.py

```diff
@@ -77,4 +79,8 @@ def config_reload(duthost, config_source='config_db', wait=120, start_bgp=True,
     modular_chassis = duthost.get_facts().get("modular_chassis")
     wait = max(wait, 240) if modular_chassis else wait
 
-    time.sleep(wait)
+    if safe_reload:
+        assert wait_until(wait, 20, 0, duthost.critical_services_fully_started), \
+                "All critical services should be fully started!"
```

Comment:
If reload source is minigraph, you should also wait for pfc_wd.

Reply:
Done.